docs/Documentation/Drivers/Classic/SnowBandOutputFiles.md

```diff
@@ -0,0 +1,35 @@
+# Snow Band Output Files
```

Comment:
I thought I had too.  In fact, what I did, was delete the `PRT_SNOW_BAND` option.